redcap_bridge/project_building.py

```diff
@@ -97,6 +97,11 @@ def build_project(project_csv, output_file=None, include_provenance=True):
     if output_file:
         with open(output_file, 'w') as f:
             f.writelines(output)
+    # Print of templates used and total number of fields
+    print("\nUsed template : ", end='')
+    for template in used_template:
+        print(template, "", end='')
+    print("with total of:", len(output), "fields", end='')
```

Comment:
You can also use the modern string formatting here... (see other string comment)